code/game/objects/structures/crates_lockers/largecrate_supplies.dm

```diff
@@ -141,7 +141,15 @@
 	desc = "A supply crate containing two boxed TL-102 mounted heavy smartguns."
 	supplies = list(/obj/item/storage/box/tl102 = 2)
 
-
+/obj/structure/largecrate/supply/weapons/standard_atgun
+	name = "\improper TAT-36 anti tank gun and ammo chest (x1, x10)"
```

Comment:
```suggestion
	name = "\improper TAT-36 anti tank gun and ammo chest (x11)"
```
if the "sidearm weapons chest (x20)" contains "eight TP-44 revolvers, and twelve TP-14 service pistols" as two items, I thought i had to make this consistent.